apps/web/components/eventtype/RequiresConfirmationController.tsx

```diff
@@ -42,11 +43,7 @@ export default function RequiresConfirmationController({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [requiresConfirmation]);
 
-  const { shouldLockDisableProps } = useLockedFieldsManager(
-    eventType,
-    t("locked_fields_admin_description"),
-    t("locked_fields_member_description")
-  );
+  const { shouldLockDisableProps } = useLockedFieldsManager(eventType, formMethods, t);
```

Comment:
We now need form methods in order to set discretely the different fields in the event type metadata